src/buffer/out/Row.cpp

```diff
@@ -65,6 +68,65 @@ constexpr OutIt copy_n_small(InIt first, Diff count, OutIt dest)
     return dest;
 }
 
+RowTextIterator::RowTextIterator(std::span<const wchar_t> chars, std::span<const uint16_t> charOffsets, uint16_t offset) noexcept :

```

Comment:
`RowTextIterator` isn't used anywhere yet, but I did use it during development as a debug aid. I expect this struct to be used in the future for things like `CHAR_INFO` reads.